recipe/meta.yaml

```diff
@@ -520,7 +520,7 @@ outputs:
         - {{ pin_subpackage('libopenvino-paddle-frontend', exact=True) }}
         - {{ pin_subpackage('libopenvino-pytorch-frontend', exact=True) }}
         - {{ pin_subpackage('libopenvino-tensorflow-frontend', exact=True) }}
-        - numpy >=1.16.6
+        - numpy <2.0.0
```

Comment:
This doesn’t seem right.

what is the big you are seeing exactly? It is possible fix the bug and avoid this pin?

Reply:
I understand that you are the openvino developer correct? Would it be appropriate to release 2024.2.1 with a patch to enable numpy 2?